content/en/docs/concepts/storage/volumes.md

```diff
@@ -1160,6 +1160,33 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} Driver. In order to use this feature, the [vSphere CSI
+Driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
+must be installed on the cluster and the `CSIMigration` and `CSIMigrationvSphere`
+features must be enabled. This also requires minimum vSphere vCenter/ESXi Version to be 7.0u1 and minimum HW Version to be VM version 15.
+
+The following StorageClass parameters from vsphereVolume plugin will not be supported by vSphere CSI driver:
+
+* diskformat
+* hostfailurestotolerate
+* forceprovisioning
+* cachereservation
+* diskstripes
+* objectspacereservation
+* iopslimit
+
+Existing volumes created using these parameters will be migrated to the vSphere CSI driver, but new volumes created by the vSphere CSI driver will not be honoring these parameters.
```

Comment:
nit: must be migrated to the vSphere CSI driver OR is a process handling the migration?
nit: will not honor these parameters

Reply:
I think "will be" is correct in the sentence.  We are talking about what will happen with CSI migration.